pyproject.toml

```diff
@@ -5,7 +5,7 @@ build-backend = "pdm.pep517.api"
 [project]
 name = "torch-dicom"
 description = ""
-requires-python = ">=3.10,<3.12"
+requires-python = ">=3.11,<3.13"
```

Comment:
Is it necessary to move the minimum from `3.10` to `3.11`? If not, it might be nice to have more flexibility.

Reply:
I'm open to leaving the pin as is, but it is worth noting that our CI pipeline doesn't test this version of Python so there isn't really a check if features requiring 3.11 are being used